tree/ntuple/v7/src/RColumnElement.cxx

```diff
@@ -57,6 +57,7 @@ ROOT::Experimental::Internal::RColumnElementBase::GetValidBitRange(EColumnType t
    case EColumnType::kSplitInt16: return std::make_pair(16, 16);
    case EColumnType::kSplitUInt16: return std::make_pair(16, 16);
    case EColumnType::kReal32Trunc: return std::make_pair(10, 31);
+   case kTestFutureType: return std::make_pair(32, 32);
    default: assert(false);
```

Comment:
```suggestion
   default: 
      if (type == kTestFutureType) 
           return std::make_pair(32, 32);
      else 
          assert(false);
```
To be used here and elsewhere to avoid the warnings:
```
case value ‘30’ not in enumerated type ‘ROOT::Experimental::EColumnType’ [-Wswitch]
```